src/parsing/Parse_contribution.ml

```diff
@@ -0,0 +1,27 @@
+(** Collect information about the project contributions from git log. *)
+
+open Common
+
+let git_json_format =
+  "--pretty=format:{\"commit_hash\": \"%H\", \"commit_timestamp\": \"%ai\", \
+   \"contributor\": {\"commit_author_name\": \"%an\", \"commit_author_email\": \
+   \"%ae\"}}"
+
+let get_git_logs () : string list =
```

Comment:
Ideally you would put that in libs/git_wrapper/Git_wrapper.ml if it's general enough and not specific to Semgrep.

Reply:
Moved the git-specific changes to `Git_wrapper.ml`